client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -52,7 +52,7 @@ const ECommerceTrialPlansPage = ( props: ECommerceTrialPlansPageProps ) => {
 						isEntrepreneurTrial ? (
 							<UpgradeButton
 								goToCheckoutWithPlan={ goToCheckoutWithPlan }
-								isEntrepreneurTrial={ isWooExpressTrial }
+								isEntrepreneurTrial={ isEntrepreneurTrial }
```

Comment:
This is not part of any comment, but I "Fixed" it wrongly on last commit and the Entrepreneur trial ended up with the red button instead the green one. But not a huge problem, since this still not available to the public.